src/pages/services/aem-cf-console-admin/api/action-bar/index.md

```diff
@@ -145,3 +145,67 @@ const guestConnection = await register({
 | Field    | Type                                                                        | Required | Description                                                                  |
 |----------|-----------------------------------------------------------------------------| ------ |------------------------------------------------------------------------------|
 | id       | `string`                                                                    | ✔️  | "id" is a "data-id" attribute of the button (please use browser developer console to find actual value). |
+
+### Get List of selected Fragments
+
+You can access data about the currently selected Content Fragments using the `host.fragmentSelections.getSelections()` method in an extension.
+
+
+## Examples:
+### Buttons conditional rendering
+
+Content Fragment Admin invokes getButtons method each time a user changes content fragments selection. Therefore, you can dynamically return different sets of buttons based on the data that has changed on the page. 
```

Comment:
Content Fragment Admin invokes the `getButtons` method each time a user changes the content fragments selection. Therefore, you can dynamically return different sets of buttons based on the data that has changed on the page.